zboxcore/sdk/sdk.go

```diff
@@ -76,17 +76,50 @@ func GetLogger() *logger.Logger {
 	return &Logger
 }
 
-// InitStorageSDK init storage sdk with wallet json and config
-func InitStorageSDK(walletJSON string, cfg conf.Config) error {
-	err := client.PopulateClient(walletJSON, cfg.SignatureScheme)
+// InitStorageSDK init storage sdk with clientJSON
```

Comment:
@cnlangzi I have been wrong here. InitStorageSDK we always passing walletJson and also in mobile Apps we calling this `Zcncore.init` where we actually passing chain config

Reply:
moved it to zcncore.Init instead of sdk.InitStorageSDK.